contrib/translations/PDE-reduction-symbolic.ipynb

```diff
@@ -48,20 +46,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from pytools import factorial\n",
-    "\n",
-    "def mi_factorial(n):\n",
-    "    return np.prod([factorial(n1) for n1 in n])"
```

Comment:
The notebook had this in, but it doesn't seem used.

I also updated it so it actually runs, `mpole_expn_reduced.derivative_wrangler` didn't seem to exist anymore. Not sure it's giving the correct results though.